wgpu-core/src/command/render.rs

```diff
@@ -1473,10 +1478,28 @@ impl Global {
                         )));
                     }
 
+                    // If this.depthLoadOp is "clear", this.depthClearValue must be provided and must be between 0.0 and 1.0, inclusive.
+                    if depth_stencil_attachment.depth.load_op == Some(LoadOp::Clear) {
+                        if let Some(clear_value) = depth_stencil_attachment.depth.clear_value {
+                            if !(0.0..=1.0).contains(&clear_value) {
+                                return Err(CommandEncoderError::InvalidAttachment(AttachmentError::ClearValueOutOfRange));
+                            }
+                        } else {
+                            return Err(CommandEncoderError::InvalidAttachment(AttachmentError::NoClearValue));
+                        }
+                    }
+
+                    let depth = PassChannel {
+                        load_op: depth_stencil_attachment.depth.load_op,
+                        store_op: depth_stencil_attachment.depth.store_op,
+                        read_only: depth_stencil_attachment.depth.read_only,
+                        clear_value: depth_stencil_attachment.depth.clear_value.unwrap_or_default()
+                    };
+
                     Some(ArcRenderPassDepthStencilAttachment {
                         view,
                         depth: if format.has_depth_aspect() {
-                            depth_stencil_attachment.depth.resolve()?
+                            depth.resolve()?
```

Comment:
Again, this part is weird, but I have plan how to make this all nicer in a follow up.